sdk/appconfiguration/app-configuration/test/public/etags.spec.ts

```diff
@@ -32,6 +32,18 @@ describe("etags", () => {
     await recorder.stop();
   });
 
+  after(async function (this: Context) {
```

Comment:
From what I understand, `deleteKeyCompletely` in the `afterEach` is used to delete keys that are made in the `beforeEach` section because we need to pass in the `keys` parameter. The cleanup in `after` should clean all the keys that are made in the tests. I do see what you mean by 2 approaches, so I'll check with @HarshaNalluru about this

Reply:
At a cursory glance, it looks like all the settings in the test have key `key` (the value set in the `beforeEach`), so I would have thought that the `afterEach` would have covered all those. I may be missing something of course.